packages/shared/routes/setup/Congratulations.svelte

```diff
@@ -87,22 +87,36 @@
                 }
             }
             const _exportMigrationLog = () => {
-                getProfileDataPath($activeProfile.id)
-                    .then((source) =>
-                        $walletSetupType === SetupType.TrinityLedger
-                            ? Platform.exportLedgerMigrationLog($migrationLog, `${$activeProfile.id}-${LOG_FILE_NAME}`)
-                            : Platform.exportMigrationLog(
-                                  `${source}/${LOG_FILE_NAME}`,
-                                  `${$activeProfile.id}-${LOG_FILE_NAME}`
-                              )
-                    )
-                    .then((result) => {
-                        if (result) {
-                            logExported = true
-                            _continue()
-                        }
-                    })
-                    .catch(console.error)
+                if ($mobile) {
+                    Platform.exportMigrationLog(`${$migrationLog}`, `${$activeProfile.id}-${LOG_FILE_NAME}`)
```

Comment:
to avoid doing a migration to test this I just faked the `migrationLog` to be something like the following and forced the continue to go through this step, just to test. 
```ts
const _migrationLog = [
    {
        bundleHash:
            'TSBGBTCBL9QVJOCJYKXCKFBUWXPYSBPVXV9GJLKXWJSQJFUAHKCLGPHMBH9SCBBPHG9KKCPUSYEQRDALB',
        trytes: [
            'DUFKJJVYANVHLGUHHXNOCKSLQYSJSJRN9OSSOFIDELOKDDKAQSVVROEBYTECRILOIYTBPAZT9ELJG9NOARADJAQOPUTADTXRBPWIDDXQUUCAWB99HNWATFMAQKDQQIDHKBBWWSGEZXV9JTIDB9AXXPZYAGTUOCD9WCMCSZILQEJXN9VAFBFPHLXIIHJUZP9JZOYVXBGSFDKOQXGXRCKQ9NHGMVUJQKNNAEMALHPWETZWTRQKGR9OJKPMWWWBEUC9PFDNJVE999CYBBBLVTIC9GFHLJZOLLHVWGLDW99CKKESAVYQXDPOUHSITOCO9XZPVIBDNXRVEH9SVKNAEKUXOXIMZFGADSE9MFOIUGOKHU9NLACJ9LPWWZTUKFTWMZDWZOSDQDY9DTSTGNTMWKBR9DKEDOZNCTMFHUHCCSKPECILNXDKJMAMAYPHVFA9TTBDWQTPDZBYMGYVZZDNBX9TROEXPMVI9SQJMUVOTC9LKUU9WXNADAFJTJFITTWUKYU9XZVPHGYSVHKIIWNZNAPDYRKYPZIDAUPOXTDINBJALEKQNGWUTAKECQDJHMIVIQGAKFUASZWCEX9FFWMRVINDABIBMWZWPWQWGYYDNDLCLMQJDXPX9TKKGJYZMCIAQZEILFCGPPKCOEZQUKMQYOPDEUKSG9GAZPFIPH9FIQFIQSTS9PMJDUCXSBVTJODOZKS9SOGGHZHBELWDMKNOWYRAJUYOWNWFEQENNSPEVZL9M9TZH9HCRPGSLTUJINZQDTAMCCPVYLZ9RMCPZW9QXDWHGKP9LXEWMDWA9HLNTKQHWWIUZBW9AHNHCSMDZGSGYPSHPIMPILROTQVENATNROZMFHKBXPILN9ATMXWYWRC9GTIYOKMLBJBWVZZYKIZAVZZ9YGCC9MUBOV9UNBVBPXAIIBXPCDCXYT9UXPCW9AFR9ZTRFUHILJUWRAATHQVAGAOJOWUZCPQGEIC9CXUBVYJD9IZWRCWMBJKJOCCSLJGVIZK9UDSLNRTHBJQXXSSELPXNGIRJOYPKIEPXQXFMZASAGYJUZFFAXKCFYFRTMCRMDSFFYFLXXGPUHHRRUDIFV99PQSEQSCFQMWDISRDTULIZLUNOIAPXVRFJRHMQESTEXTOFJZSGDCGRWIUUICECNPOI9SEQEKPKTAFHXIZYAKHUOQGEU9WAMH9G9XJKINC9MAVGBXCFWOOPDWEUQEPZHCWBAOADPXUBSFKVUFGWQRIHEANQKT9WJRDSDFUNCDERLBGNTVTVHMUTUPBBMQIAPXNQZZSWGNBDDBJYMKYAUADNXIRYCLDKPKUFQSC9OEZHRNNXLVTWIZDAJHKTWARHGF9YSZYDSJZSMKVCEXSEFTXQADOLJTIBFIJIZBIKUOAGYXIUUJNITBGSYRQJWOFBZFNU9WKTUQCIMVNHGJWEHCFRXJTXFNDHBKMXBGBIPPHCNBQVEEAHCWQNTR9EMKDPHTOPGLBMCDPWOFBZRLOEICMXOVOEIVYMWQZVDYMSZVYIPWXUT9GYB9MRDLIVOW9FHIFKZ9YWACNRHVVVRBEIS9SRPZIEFDOBRYCZNAOWKW9DGZXUUCKIZUHUZYRPQEQQO9WJXPIHVAFLVGOGWJ9NMQTZZYGHDY9JUDSXMPZPCRRSUNXGTRXPGGMKASKRGBFCXCNXGYGYXEOILREJOHEOGHLXDG9PSZFCRMGESSXLBFCVWZVJHCNTRWREPBAZMQTHMFSECRMJYHQXW9SDZCIWCXCRFLNMVYXNDCDKWDXQHGDMEBPAS9BHATEQLCBYLDKGHAMTWDPYLEN9WSECXOPDSKLSXJDOUKPSFIPNHYPBYMSGPKPKKZUKFDJMNBWVMXFOZFZGJTFNPQXHEYCDYOITNCSNDZHRTTZGFIGUZODVKJ9PTGIPDQJVESVQHERJZAATQFPJLYSUOBLZYPFHXRALBROWW9NWIMYOKIUHHTRHHQHZKCUUEVFCQON9BRMSHRIUDHAYGQ9FPSSYOFDMTVWEM99ZSIXBLLUDUTBQGCPEQGPYCHWIJBV9NEOTOMQTATZIKE9VUBFVWJNLM9CMTXMHMXNFMCZUNAJLJVJFFZNW9MBOYDIMBBCQIIEYVRTXCCWEBABYE9VVKHNGJLWULPMCJJZFNM9NZYRK9NYDYXDWLQENVLNASYISYTJLSYYXTDTRJDPQHLEBVVAYGUWXMXJXZUTOSZSJWEUAIPNGUMDO9MHHNJYYYGYBOFMPWDTJTIVFKHOKJPKGYMUYGMAEQKTVKV9IMHASLUSD999999999999999999999999999999999999999999999999999999TMGVSHD99B99999999B99999999TSBGBTCBL9QVJOCJYKXCKFBUWXPYSBPVXV9GJLKXWJSQJFUAHKCLGPHMBH9SCBBPHG9KKCPUSYEQRDALB999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999XWVVPGVKORIXOE9999999999999999999999',
            'HZTRHTZKTMJGRKWBPQPKFRCFFQFRGVSQFNFMWUOVUBVELIHFFVKLZ9FHFIMPCGBMCVGBBAWJPYNWBXTZWSSPZXWFQLVZMMBOIIZYURBZO99QA9DWFEAF9FWAO9GKILGAGOUHGPLVJXQHXGLMUZYRXGPOIPPBMUIAPWNPJMBJILDAPAYMFOPQPGPCHPVPHK9EKPKCUEGARRFEXVTBOACPUIQLFHGBCYDCJXQLCUPXXJMJQLJSUMDSUUAVJUKBJNEXMXGTSRIG9PNOMTXLFAEXJJ9ZSANRLGBH9HMOPYQQVVQWYPNRUZDIWAIVXDBJJXYABEUXRWIQICN9EOOBSQUP9OYPKLQZRACETZEBQSUULKZEEURYNOJSCDCETAIPKHLXBKMVKIHZHCHWJADSTGKLDLDTEQXDFZOM9XAUKVDRK9IOPXMJTDCOMRKGENXQAEGPYEQXMQKCVZEPDIHXEKSZVUIJQFJZYPZSHAITWDPMKNTPWHNPLSHOMPRYNCEKLOIOTNAWDEPGUKDNAYJWIQLPTPPIWKCUNQJUJSRULYN9MLGKYKLZUOQVJMWPDJRSQVDBCCQSWYDSCHZGJWM9GFPDLRLDMAVWBUKEGORLIBHKRVRXRXYZKHEMHBVOXOPWTJQTPOJ9PYECEBLZNNNRJRHPQCZJMEFUIEVCTAOPCOJRUMDCAHXRNEJZFLAFEWDNUNFWWSZHSFLZQWIGBVOPMUHUBCLZ9QVVCEDNJZKSNAROUTIHZRKRKUDVOZDJNNLWFTYIEKYPIUUONDQJOINSVJANS9JRKIICWSQVEUNNFLCHADJN9GXALVULAGYVIWMNEPQHQUZNDYHRHGWN9OFIQRRVWAKEFDHTZM9C9GSYIIAMDPUZBIGOIJM99IKCISCHDFKPLXKQ9FFKCNLXSELZEMYCGM9LUHFARCEQZCSOLTQKQWAASE9TRJCIVEETHPJNZFTNQEYWKMMQXBXDEFYDEWNVKRJVIMKQYOPKBRLGPVIARRSKXYTEEGOLVSDIW9WIFLYEFWWERODWYYRADGUENHBZUVOZCBDKAFFLAKCNEOUJQSIDDWHKYZAEKEUJF9IWQWYSYZBKONPSMZJYKVEHDAIVZWRDCNPFWFPHBAOSTBUTLMHKRAJHZGYSHFMQPYJQJGRUDULYQERZ9OXSWJDAAEBQLEWIDUJAKOZ9LYQXAVTTIWONUUOAC9HNVDFYUBNLZFALLXOJLLJMIGIDERUDECZ9HYN9WETXTSEYQTKLD9YAUVXIBJOKQYUKJXNXLWIREXEZFEIOQMCBN9AWTBAA9CJFFDUZBUCSALHTFQ9ZRPCMGVDRYNKKRCGNTORGIJCWWLLWLW9XWW9UYRKWYSGKFGFRCFWJLFDMFXGUWTILETBTVQLRZLKUKWXEISPYCQKPPLGMCVPZUAXFU9HWKJWKNVLJB9QZGVIUWTGCWEBJKZNPHM9MJRBOWAVM9NAVERACPBQGCLAYPQFBFLYOTADFDLEAKSZVKRTPDJUENUJTIBDZPGCTBIRAWMKSAZYYRBGRPBCNABDPVKJZPERHJGWRSECOZAAQHPLBWEPWKKINKSJFAG9IBBRESLYYLAYDTBRHGBIINKIELACIMRAUMGEFMOYCDACGRICMOZ9SX9ZJQIVTFJUJFFFJUTCOXZKSBYQOHLGTBBBGKHMZXEOZIDVTP9NQBV9KP9HSOYEIQBHSZTSURSUTXWYLPBXWPVYXTRUIMWADR99MDAQGEDWGCRCBSULWGNDPWKUVLNZVIMLWDRGRPCCHKP9HHXJNA9ISEYNNKNWQCJLTJL9FIKOXEOJNCOSYBLPWBSFVBCCGQIWQFPPJT99DEEWIMJOQOLXMTVOXGMVWQCKTXSDNAQERTYMHJQHRAWTKOCYOZUFCSX9ULAQJMSXXKXXYMGPFYGVEUXXXRYQFLGFDHATIMOGVUEHLTRHQLEYWUTFDSJFQPITHBRWSCUYDQYBOGFNEMPWXXBBWCXGAQEVAI9IDZFTXJTVBRUGIEWGCDOJGZWCZOPZDMCMNWATAJBF9SG9PXZJQVDFIQDAUKZNPHVVRSJIPKXOOKRCAAANUHLEUDLPY9ECXYZ9UGQCBPKSZBUFNKSXFG9KIYGBDXTFPUKOFCIHDAVFNTCVPXWPZKGMHUSQFDCCHBRVRYFVEQ9QJSIEHEUKF9OK9YAHSBUCOPBMWOPDXMXJXZUTOSZSJWEUAIPNGUMDO9MHHNJYYYGYBOFMPWDTJTIVFKHOKJPKGYMUYGMAEQKTVKV9IMHASLUSDLJUEX9999999999999999999999999999999999999999999999999TMGVSHD99A99999999B99999999TSBGBTCBL9QVJOCJYKXCKFBUWXPYSBPVXV9GJLKXWJSQJFUAHKCLGPHMBH9SCBBPHG9KKCPUSYEQRDALB999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999XWVVPGVKORIXOE9999999999999999999999',
            '999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999TRANSFERJYJVUXLBBYBYZWRXNE9AEAMVXZVDAYDYWASDPESYLCTCXAT9TXPEIYEAH9QZI9PBSALBEZFX9OQFVC9999999999999999999999J99999999999999999999999999TMGVSHD99999999999B99999999TSBGBTCBL9QVJOCJYKXCKFBUWXPYSBPVXV9GJLKXWJSQJFUAHKCLGPHMBH9SCBBPHG9KKCPUSYEQRDALB999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999XWVVPGVKORIXOE9999999999999999999999',
        ],
        receiveAddressTrytes:
            'TRANSFERJYJVUXLBBYBYZWRXNE9AEAMVXZVDAYDYWASDPESYLCTCXAT9TXPEIYEAH9QZI9PBSALBEZFX9',
        receiveAddressBech32: 'atoi1qr2g822zenxf8fn6rvsgdcn8e08pwerucfw55x8e4p7dxgqgmvyjk9awz5p',
        balance: 1500000,
        timestamp: '2022-08-10 10:51:56.885977072 UTC',
        spentAddresses: [],
        spentBundleHashes: null,
        mine: false,
        crackability: null,
    },
]
```

Unfortunately, when I send the migration log this is what I get when I open it `[object Object]`

Reply:
pushed a fix